rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -435,58 +438,53 @@ public void registerBtcTransaction(
     private void registerSvpFundTx(BtcTransaction btcTx) throws IOException {
         logger.info("[registerSvpFundTx] This is an svp fund tx {}", btcTx.getHash());
 
-        processPegoutOrMigration(btcTx); // Need to register the change UTXO
+        registerNewUtxos(btcTx); // Need to register the change UTXO
 
+        // If the SVP validation period is over, SVP related values should be cleared in the next call to updateCollections
+        // In that case, the fundTx will be identified as a regular peg-out tx and processed via #registerPegoutOrMigration
+        // This covers the case when the fundTx is registered between the validation period end and the next call to updateCollections
```

Comment:
What if there is no powpeg call to `updateCollections`(let's think hypothetically, all powpeg running has the option disabled), and then a new federation is committed? What would happen with those uncleared values? Would it be worth it to have this test case like this?

Reply:
reaching the commitFederation point means we have already cleared all the svp values